<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyAppointments</class>
 <widget class="QDialog" name="MyAppointments">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>My Appointments</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
        stop:0 #e3f2fd, stop:1 #ffffff);
}</string>
  </property>
  <widget class="QFrame" name="frameHeader">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
        stop:0 #1976d2, stop:1 #42a5f5);
    border: none;
}</string>
   </property>
   <widget class="QLabel" name="lblTitle">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>1100</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 36px;
    font-weight: bold;
    color: white;
    background: transparent;
    qproperty-alignment: AlignCenter;
}</string>
    </property>
    <property name="text">
     <string>üìã My Appointments</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frameFilters">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>1100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
    background-color: white;
    border-radius: 12px;
    border: 2px solid #e0e0e0;
}</string>
   </property>
   <widget class="QLabel" name="lblFilterBy">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>25</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    font-size: 15px;
    font-weight: 600;
    color: #37474f;
    background: transparent;
}</string>
    </property>
    <property name="text">
     <string>Filter by:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboFilterStatus">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>201</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
    background-color: #fafafa;
    border: 2px solid #e0e0e0;
    border-radius: 8px;
    padding: 8px 12px;
    font-size: 14px;
    font-weight: 600;
    color: #212121;
}
QComboBox:hover {
    border: 2px solid #1976d2;
}
QComboBox::drop-down {
    border: none;
    width: 30px;
    background: #1976d2;
    border-top-right-radius: 6px;
    border-bottom-right-radius: 6px;
}
QComboBox::down-arrow {
    image: none;
    border: 4px solid transparent;
    border-top: 4px solid white;
    margin-right: 12px;
}
QComboBox QAbstractItemView {
    background-color: white;
    border: 2px solid #1976d2;
    selection-background-color: #e3f2fd;
    selection-color: #1565c0;
}</string>
    </property>
    <item>
     <property name="text">
      <string>All Appointments</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Upcoming</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Completed</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cancelled</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnRefresh">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>20</y>
      <width>170</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-size: 14px;
    font-weight: 600;
    color: white;
    background-color: #1976d2;
    border: none;
    border-radius: 8px;
    padding: 8px;
}
QPushButton:hover {
    background-color: #1565c0;
}
QPushButton:pressed {
    background-color: #0d47a1;
}</string>
    </property>
    <property name="text">
     <string>üîÑ Refresh</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableWidget" name="tableAppointments">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>1100</width>
     <height>470</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableWidget {
    background-color: #ffffff;
    alternate-background-color: #f5f9ff;
    gridline-color: #d0e3f7;
    selection-background-color: #90caf9;
    selection-color: white;
    border: 2px solid #1976d2;
    border-radius: 10px;
    font-size: 14px;
}
QTableWidget::item {
    padding: 10px;
    border-bottom: 1px solid #e0e0e0;
}
QTableWidget::item:selected {
    background-color: #64b5f6;
    color: white;
}
QHeaderView::section {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1,
        stop:0 #1976d2, stop:1 #1565c0);
    color: white;
    font-weight: bold;
    font-size: 14px;
    padding: 10px;
    border: none;
    border-right: 1px solid #0d47a1;
}
QHeaderView::section:first {
    border-top-left-radius: 8px;
}
QHeaderView::section:last {
    border-top-right-radius: 8px;
    border-right: none;
}
QScrollBar:vertical {
    background: #f5f5f5;
    width: 12px;
    border-radius: 6px;
}
QScrollBar::handle:vertical {
    background: #90caf9;
    min-height: 30px;
    border-radius: 6px;
}
QScrollBar::handle:vertical:hover {
    background: #64b5f6;
}
QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    height: 0px;
}</string>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="columnCount">
    <number>6</number>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Doctor</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Date</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Time Slot</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Status</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Actions</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnViewDetails">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>720</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    font-size: 16px;
    font-weight: bold;
    color: white;
    background-color: #42a5f5;
    border: none;
    border-radius: 10px;
}
QPushButton:hover {
    background-color: #1976d2;
}
QPushButton:pressed {
    background-color: #1565c0;
}</string>
   </property>
   <property name="text">
    <string>üëÅÔ∏è View Details</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelAppointment">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>720</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    font-size: 16px;
    font-weight: bold;
    color: white;
    background-color: #ff5252;
    border: none;
    border-radius: 10px;
}
QPushButton:hover {
    background-color: #f44336;
}
QPushButton:pressed {
    background-color: #d32f2f;
}</string>
   </property>
   <property name="text">
    <string>‚ùå Cancel Appointment</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReschedule">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>720</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    font-size: 16px;
    font-weight: bold;
    color: white;
    background-color: #ff9800;
    border: none;
    border-radius: 10px;
}
QPushButton:hover {
    background-color: #f57c00;
}
QPushButton:pressed {
    background-color: #e65100;
}</string>
   </property>
   <property name="text">
    <string>üîÑ Reschedule</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>720</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    font-size: 16px;
    font-weight: bold;
    color: white;
    background-color: #78909c;
    border: none;
    border-radius: 10px;
}
QPushButton:hover {
    background-color: #546e7a;
}
QPushButton:pressed {
    background-color: #37474f;
}</string>
   </property>
   <property name="text">
    <string>‚Üê Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
